.docker/images/govcms8/settings/settings.php

```diff
@@ -34,9 +34,9 @@
 if (file_exists($contrib_path . '/fast404/fast404.inc')) {
   include_once $contrib_path . 'fast404/fast404.inc';
 }
-$settings['fast404_exts'] = '/^(?!robots)^(?!sites\/default\/files\/private).*\.(?:png|gif|jpe?g|svg|tiff|bmp|raw|webp|docx?|xlsx?|pptx?|swf|flv|cgi|dll|exe|nsf|cfm|ttf|bat|pl|asp|ics|rtf)$/i';
+$settings['fast404_exts'] = '/^(?!robots)^(?!sites\/default\/files\/private).*\.(?:png|gif|jpe?g|svg|tiff|bmp|raw|webp|pdf?|docx?|xlsx?|pptx?|swf|flv|cgi|dll|exe|nsf|cfm|ttf|bat|pl|asp|ics|rtf)$/i';
```

Comment:
```suggestion
$settings['fast404_exts'] = '/^(?!robots)^?!\/system\/files).*\.(?:png|gif|jpe?g|svg|tiff|bmp|raw|webp|pdf?|docx?|xlsx?|pptx?|swf|flv|cgi|dll|exe|nsf|cfm|ttf|bat|pl|asp|ics|rtf)$/i';
```

Fast 404 is blocking access to private document files we should update this so that it allows system/files.

For example:

```
$ http -h http://rcdisability.docker.amazee.io/system/files/2020-02/file-sample_100kB.docx                                                      
HTTP/1.1 404 Not Found
Cache-Control: must-revalidate, no-cache, private
```

Reply:
Sounds good, I'll adjust this shortly.